src/srcdocs/makedox.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: © Fossology contributors
```

Comment:
File makedox looks like a bash script. So it will be better to put the licensing information there itself using `#`

Reply:
Done.